### PR TITLE
Merge python quote examples

### PR DESCRIPTION
**Pull request by ATGE**
#449

### CODE REVIEW

File: content/python/quotes.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+### Setup
+All the functions defined in this article will be part of this `Example` class. Which only sets up the SoftLayer Client, and configures the debugger, which allows you to see the exact API calls being made.
+
+```python
+import SoftLayer
+from SoftLayer.managers.ordering import OrderingManager
+from pprint import pprint
+
+class Example:
```
Comment (caberos) [inline content/python/quotes.md:20]: change to QuoteExamples

File: content/python/quotes.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
+---
+title: "Working with Quotes"
+description: "A few examples on interacting with Quotes"
+date: "2021-02-26"
+classes: 
+    - "SoftLayer_Account"
+    - "SoftLayer_Billing_Order_Quote"
+tags:
+    - "quotes"
```
Comment (allmightyspiff) [inline content/python/quotes.md:9]: add the tag `ordering`  as well

File: content/python/quotes.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                  "sshKeys": [{"sshKeyIds": [123456]}]
+                  }
+        if verify:
+            order = self.mgr.verify_quote(quote_id, extras)
+        else:
+            order = self.mgr.order_quote(quote_id, extras)
+        pprint(order)
+
+```
+## Saving a Quote
```
Comment (allmightyspiff) [inline content/python/quotes.md:112]: Add this explanation for why you would want to save a quote:


"Quotes generally expire after a few days, saving a quote will prevent the system from removing it, letting you order from this quote in the future"

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Just a few minor changes needed, looks great otherwise, thanks.

File: content/python/quotes.md (hunk 1 above)
Comment (ATGE) [reply content/python/quotes.md]: done

File: content/python/quotes.md (hunk 2 above)
Comment (ATGE) [reply content/python/quotes.md:9]: added

File: content/python/quotes.md (hunk 3 above)
Comment (ATGE) [reply content/python/quotes.md:114]: done

**Review by caberos — APPROVED** [review]
LGTM

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**